embedded_rtic/examples/sonars.rs

```diff
@@ -0,0 +1,440 @@
+//! Defines a simple distance measurement example.
```

Comment:
Add a bit better docs for the example these are relevant to the sonar example not the sonars example